scirpy/io/_io.py

```diff
@@ -136,6 +136,15 @@ def _read_10x_vdj_json(path: Union[str, Path], filtered: bool = True) -> AnnData
         chain["is_cell"] = cell["is_cell"]
         chain["high_confidence"] = cell["high_confidence"]
 
+        # additional cols from CR6 outputs: fwr{1,2,3,4}{,_nt} and cdr{1,2}{,_nt}
+        fwrs = [f"fwr{i}" for i in range(1, 5)]
+        cdrs = [f"cdr{i}" for i in range(1, 3)]
+
+        for col in fwrs + cdrs:
+            if col in cell.keys():
+                chain[col] = cell[col].get("aa_seq") if cell[col] else None
+                chain[col + "_nt"] = cell[col].get("nt_seq") if cell[col] else None
```

Comment:
According to the [rearrangement standard](https://docs.airr-community.org/en/latest/datarep/rearrangements.html)

the fields are `cdr1` for nt and `cdr1_aa` for aa